### ISSUE TITLE
Slow mouse cursor with 'retina' resolutions in macOS

### ISSUE THREAD

**Issue by almeath** [joncampbell123/dosbox-x#1470]
**Describe the bug**
When setting fullresolution to the native display resolution on a 27 inch iMac (5120x2880) to provide the sharpest possible scaled graphics, the mouse cursor slows down to about 50% of its expected speed (compared to DOSBox 0.73-x with the equivalent settings).

**To Reproduce**
On an iMac with a 27 inch retina display (5120x2880) set the fullresolution in the DOSBox.conf file to 5120x2880 and leave the sensitivity setting on the default value of 100

**Expected behavior**
The mouse cursor will move at the same speed as when fullresolution is set to 'desktop' or the pixel doubled '2560x1440' resolution.

**Environment (please complete the following information):**
 - Operating system: macOS Mojave 10.14.6 with all security updates to April 2020
 - DOSBox-X release version: 0.83.0 (SDL1)
 - Used configuration: 
[sdl]
fullscreen        = true
fulldouble        = true
fullresolution    = 5120x2880
windowresolution  = original
output            = openglnb
autolock          = false
autolock_feedback = beep
sensitivity       = 100
mouse_emulation   = locked
waitonerror       = true
priority          = higher,normal
mapperfile        = mapper-0.82.23.map
usescancodes      = false
overscan          = 0
titlebar          = 
showmenu          = true

**Additional context**
The equivalent settings in DOSBox 0.74-3 do not produce a slow down in the mouse cursor, even when forcing output to full retina resolution of 5120x2880.

**Comment by almeath** [comment]
This is a follow up to confirm I still see this behavior in the latest build for macOS on my iMac with a “retina” screen.

In vanilla DOSBox the mouse cursor tracking is normal at any resolution I set. In DOSBox-X the mouse cursor tracking is substantially slower, about half as fast as it should be, and that is with sensitivity already set to 100. 

The tracking only works at normal/full speed if I set my full screen resolution to a “non-retina” / native size such as half my native resolution; 2560x1440.

Since none of the mouse settings have any impact, this appears to be a bug with the mouse tracking speed on Macs with retina displays.
